pkg/cli/push.go

```diff
@@ -23,10 +23,12 @@ import (
 
 var pushDir string
 var pushDryRun bool
+var pushDeleteNonProvidedTeams bool
 
 func init() {
 	pushCmd.PersistentFlags().StringVarP(&pushDir, "push-directory", "d", ".", "Directory of content to push to Insights.")
 	pushCmd.PersistentFlags().BoolVarP(&pushDryRun, "dry-run", "z", false, "Explain what would be pushed to Insights, without making changes.")
```

Comment:
Looks like this should be updated `Explain what would be pushed to Insights, without making changes.`